awx_collection/plugins/modules/tower_job_wait.py

```diff
@@ -130,14 +137,14 @@ def main():
         )
 
     # Attempt to look up job based on the provided id
-    job = module.get_one('jobs', **{
+    job = module.get_one(job_type, **{
         'data': {
             'id': job_id,
         }
     })
 
     if job is None:
-        module.fail_json(msg='Unable to wait on job {0}; that ID does not exist in Tower.'.format(job_id))
+        module.fail_json(msg="Unable to wait on " + job_type.rstrip("s") + ' {0}; that ID does not exist in Tower.'.format(job_id))
```

Comment:
This is super nitpicky, but can we use either `'` or `"` vs both here?

Reply:
I addressed this issue. I will address the others one at a time.